CDP4ParameterSheetGenerator/Assemblers/OptionSheet/OptionSheetRowAssembler.cs

```diff
@@ -120,6 +120,29 @@ public void Assemble()
             foreach (var nestedElementExcelRow in nestedElementExcelRows)
             {
                 var deeprows = nestedElementExcelRow.GetContainedRowsDeep();
+
+                foreach (var itm in deeprows)
```

Comment:
Do not shorten variable names. If `itm` meand `item` then use that. But in principle you should use something more descriptive than `item` as well.